components/script/dom/countqueuingstrategy.rs

```diff
@@ -122,7 +122,10 @@ pub(crate) fn extract_high_water_mark(
 /// If the size algorithm is not set, return a fallback function which always returns 1.
 ///
 /// <https://streams.spec.whatwg.org/#make-size-algorithm-from-size-function>
-pub(crate) fn extract_size_algorithm(strategy: &QueuingStrategy) -> Rc<QueuingStrategySize> {
+pub(crate) fn extract_size_algorithm(
+    strategy: &QueuingStrategy,
+    _can_gc: CanGc,
```

Comment:
Hi @jdm. Can you please tell me if this is what you had in mind when you updated https://github.com/servo/servo/issues/34573? Thanks.